hack/non-olm-install/README.md

```diff
@@ -7,19 +7,25 @@
 The `install-gitops-operator.sh` script supports two methods of installation.
 1. Using operator and component images set as environment variables (default method)
 2. Derive the operator and component images from the `ClusterServiceVersion` manifest present in the operator bundle
```

Comment:
In which use case would a user need to install the operator via the operator-bundle?

Reply:
We can use `BUNDLE_IMG` to extract the component build URLs from inside the `ClusterServiceVersion` manifest inside the bundle.